### PR TITLE
Fix: Added missed 'authtoken' field

### PR DESCRIPTION
**Pull request by akshayhavile**
Added missing "authtoken" field in
1. values.yaml
2. AKO operator files

### CODE REVIEW

**Comment by akshayhavile** [comment]
build ako

**Review by monotosh-avi — APPROVED** [review]

**Comment by akshayhavile** [comment]
build ako

File: ako-operator/helm/ako-operator/templates/secret.yaml [hunk 1]
```diff
@@ -7,6 +7,9 @@ type: Opaque
 data:
   username: {{ .Values.avicredentials.username | b64enc }}
   password: {{ .Values.avicredentials.password | b64enc }}
+  {{ if .Values.avicredentials.authtoken  }}
+  authtoken: {{ .Values.avicredentials.authtoken | b64enc }}
+  {{ end }}
```
Comment (aaha97) [inline ako-operator/helm/ako-operator/templates/secret.yaml:12]: need to make the password optional too as user will provide with only 1 of the two fields

Comment (akshayhavile) [reply ako-operator/helm/ako-operator/templates/secret.yaml:14]: Thanks. Addressed.

**Comment by akshayhavile** [comment]
build ako

**Review by aaha97 — APPROVED** [review]

**Review by monotosh-avi — APPROVED** [review]

**Comment by monotosh-avi** [comment]
Should we cherry-pick this to 1.5.2 ?

**Comment by akshayhavile** [comment]
> Should we cherry-pick this to 1.5.2 ?

Yes. I will do it.

**Pull request merged**